app/views/shared/_form.html.erb

```diff
@@ -0,0 +1,65 @@
+<div class="container w-50">
+  <%= form_with model: user, url: url do |form| %>
```

Comment:
same for this file add proper validation for the user, HTML validation is okay but what if in some reason HTML validations fail? Please learn why backend validation is necessary